data/openstudio_standards_data/test/test_database.py

```diff
@@ -44,5 +60,47 @@ def test_create_database():
     res = cur.fetchall()
     assert len(res) == 0, f"Foreign key issue: {res}"
 
-    # Close DB
+    # Create a copy of original JSON files
+    if os.path.isdir("./original_database_files"):
+        shutil.rmtree("./original_database_files", ignore_errors=True)
+    shutil.copytree("./database_files", "./original_database_files")
+
+    # Export data to JSON and CSV files
+    export_openstudio_standards_database_to_json(conn, save_dir="./database_files/")
+    export_openstudio_standards_database_to_csv(conn, save_dir="./database_files/")
     conn.close()
```

Comment:
Do we need to close the connection here? It seems the conn is used in line 84 and 87

Reply:
the `conn` on line 71 is different than the one on lines 85 and 88.